prometheus_api_client/prometheus_connect.py

```diff
@@ -5,14 +5,15 @@
 import sys
 import json
 import logging
+import numpy
```

Comment:
this is not present in https://github.com/AICoE/prometheus-api-client-python/blob/master/requirements.txt so im guessing we'll have to add it there